doc/test-upgrade.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# This script is used to test the upgradeversion script without spinning up the
+# whole OpenSight stack. It creates a new (outdated) PostgreSQL database,
+# attempts to upgrade it and checks if the version file was updated correctly.
+
+set -euo pipefail
+
+cleanup() {
+  sudo rm -rf "${tempdir}"
+}
+
+trap cleanup ERR
+
+tempdir=$(mktemp -d -t opensight-postgres-test-XXXXX)
+
+sudo chown -R 999:999 "${tempdir}"
+
+# Start a PostgreSQL container with an outdated version
+docker run -it --rm --user 999:999 -e POSTGRES_PASSWORD=password -e POSTGRES_DB=keycloak -e POSTGRES_USER=keycloak -v "${tempdir}":/var/lib/postgresql/data packages.greenbone.net/opensight/opensight-postgres:16.6
+
+# Upgrade the database
+docker run -it --rm --user 999:999 -e POSTGRES_PASSWORD=password -e POSTGRES_DB=keycloak -e POSTGRES_USER=keycloak -v "${tempdir}":/var/lib/postgresql/data packages.greenbone.net/opensight/opensight-postgres:17.6 upgradeversion inplace
```

Comment:
Here are also fixed tag in the docker tags...

And you may want to create variables for `/var/lib/postgresql/data` and `packages.greenbone.net/opensight/opensight-postgres`?

Reply:
Changed as requested in commit 834b16005d060f396cdf08b187810ccef4cbe48d
I did not change the mount locations inside the container, but I split the lines to make it easier to see the usage